components/script/dom/webglrenderingcontext.rs

```diff
@@ -1039,6 +1039,147 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     }
 
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.10
+    fn Uniform2fv(&self,
+                  _cx: *mut JSContext,
+                  uniform: Option<&WebGLUniformLocation>,
+                  data: Option<*mut JSObject>) {
+        let data = match data {
+            Some(data) => data,
+            None => return self.webgl_error(InvalidValue),
+        };
+
+        if let Some(data) = array_buffer_view_to_vec_checked::<f32>(data) {
+            if data.len() < 2 {
+                return self.webgl_error(InvalidOperation);
+            }
+
+            self.Uniform2f(uniform, data[0], data[1]);
```

Comment:
Yep, that's true... We also do this in other parts of the code, so we should add `Uniformxy` messages that receive a `Vec<y>` and allow that.

If you want to do it for this PR, that would be just great, but if you'd prefer to land it as is and leaving it as a followup it'd be fine too.